lib/fastlane/plugin/revenuecat_internal/helper/versioning_helper.rb

```diff
@@ -225,6 +248,25 @@ def self.get_type_of_bump_from_commits(commits, github_token, rate_limit_sleep,
         end
         type_of_bump
       end
+
+      private_class_method def self.native_releases_links(github_token, phc_version, versions_file_path)
+        versions_latest_release = File.readlines(versions_file_path)[2].gsub(/[[:space:]]/, '').split('|')
+        previous_ios_version = versions_latest_release[IOS_VERSION_COLUMN]
+        previous_android_version = versions_latest_release[ANDROID_VERSION_COLUMN]
+
+        new_android_version = Helper::UpdateHybridsVersionsFileHelper.get_android_version_for_hybrid_common_version(phc_version)
+        UI.message("Obtained android version #{new_android_version} for PHC version #{phc_version}")
+        new_ios_version = Helper::UpdateHybridsVersionsFileHelper.get_ios_version_for_hybrid_common_version(phc_version)
+        UI.message("Obtained ios version #{new_ios_version} for PHC version #{phc_version}")
+
+        android_releases = Helper::GitHubHelper.get_releases_between_tags(github_token, previous_android_version, new_android_version, 'purchases-android')
+        ios_releases = Helper::GitHubHelper.get_releases_between_tags(github_token, previous_ios_version, new_ios_version, 'purchases-ios')
```

Comment:
Should we have a `constants.rb` file or something like that to hold these repo names?

Reply:
Yes yes, I agree with that idea 😊 This might be a good place to also put that `PR_TITLE` constant as well